Site/CuratedContentEditorialWorkflow.md

```diff
@@ -60,6 +60,26 @@ Reference:
 1. An **editorial assistant** merges the PR and marks the metadata for publication. The PR is
    moved to the *Done* column.
 
+# Criteria for Content
+
+1. **In-scope Content** ( The exact scope of relevant content will evolve as we learn about community interests. )
+   * General issues in productivity and sustainability that overlap with common challenges faced in the CSE software community.
+   * General tools for productivity and sustainability that can be widely used by CSE developers.
+   * Characterization of challenges and solutions that may be particularly valuable to the CSE community.
+
+1. **Highlight Connections to CSE**
+   * How the topic intersects with themes that are of particular interest to CSE, for example MPI, Fortran, C++, architectures, modeling and simulation.
+   * Why the topic could be of particular interest to CSE.
+
+1. **Ease of Adoption and Use**
+   * Does it address how easy or hard it is to benefit from the topic?
```

Comment:
These both seem like statements about *how* a proposed piece of content should handle a topic, the kinds of stuff the content should include in order meet this criteria. 

And, I am all for ensuring our content *goes the extra mile* in making content that is easy to quickly read and accessible to the larger HPC/CSE community. I think content that has these qualities  can be more useful to our readers. But, I don't think all content should be *required* to have these qualities. I am not even sure all content we currently host *can* have these qualities.

Reply:
I think we should consider these two, maybe make it optional, but it will give the reviewer food for thought. @markcmiller86 How would you like to change this?

Reply:
So, I think I am off-base on my comment here. I think I was reading it with a different thinking cap. I think these are fine criteria to include for curated content. If I can think of others, I will try to add.